runtime/rust/src/icp/events.rs

```diff
@@ -0,0 +1,43 @@
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use std::fmt::Debug;
+
+use crate::Result;
+
+// #[async_trait]
+// pub trait Publisher: Debug + Clone + Send + Sync {
+//     async fn publish(&self, event: &(impl Serialize + Send + Sync)) -> Result<()>;
+// }
+
+/// A [EventPlane] is a component that can publish and/or subscribe to events on the event plane.
```

Comment:
Should this say "on the event plane" or "on <something else>"? Seems weird to refer to itself - "An event plane publishes to events on the event plane"
```suggestion
/// A [EventPlane] is a component that can publish and/or subscribe to events.
```